src/override_type/acs_webchat-chat-adapter.d.ts

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// typings/acs_webchat-chat-adapter.d.ts
+declare module 'acs_webchat-chat-adapter' {
+    export function createACSAdapter(...args: any[]): any;
```

Comment:
Let's fix this, shared you a typing offline.

Reply:
since we are loading now from node module, which doesnt contains its own type, it seems like the only way to avoid the error is by declaring the module.

This technique doesn allow to use our own type definition, since this is a blocked scope function, so it cant be extended. 

We can explore more in future tasks.